rest_framework_nested/relations.py

```diff
@@ -26,7 +19,7 @@ class NestedHyperlinkedRelatedField(rest_framework.relations.HyperlinkedRelatedF
 
     def __init__(self, *args, **kwargs):
         self.parent_lookup_kwargs = kwargs.pop('parent_lookup_kwargs', self.parent_lookup_kwargs)
-        super(NestedHyperlinkedRelatedField, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
```

Comment:
That part can stay. It helps typing tools.
```suggestion
        super(NestedHyperlinkedRelatedField, self).__init__(*args, **kwargs)
```

Reply:
I don't mind keeping these, but I doubt it's necessary really.

Which typing tools? I would think they are capable of finding the base class from the `class ...(...):` definition.

Reply:
Looks like _at least_ VSCode have trouble detecting upper classes in some cases, if the middle classes do not put its names in the `super()`.

Well, lets have this merged and revert back if some complains arise :+1: